core/src/main/java/org/testcontainers/containers/DockerComposeContainer.java

```diff
@@ -576,27 +493,6 @@ public ContainerisedDockerCompose(List<File> composeFiles, String identifier) {
         addEnv("DOCKER_HOST", "unix:///docker.sock");
         setStartupCheckStrategy(new IndefiniteWaitOneShotStartupCheckStrategy());
         setWorkingDirectory(containerPwd);
-
-        String dockerConfigPath = determineDockerConfigPath();
-        if (dockerConfigPath != null && !dockerConfigPath.isEmpty()) {
-            addFileSystemBind(dockerConfigPath, DOCKER_CONFIG_FILE, READ_ONLY);
-        }
-    }
-
-    private String determineDockerConfigPath() {
-        String dockerConfigEnv = System.getenv(DOCKER_CONFIG_ENV);
-        String dockerConfigProperty = System.getProperty(DOCKER_CONFIG_PROPERTY);
-        Path dockerConfig = Paths.get(System.getProperty("user.home"), ".docker", "config.json");
-
-        if (dockerConfigEnv != null && !dockerConfigEnv.trim().isEmpty() && Files.exists(Paths.get(dockerConfigEnv))) {
-            return dockerConfigEnv;
-        } else if (dockerConfigProperty != null && !dockerConfigProperty.trim().isEmpty() && Files.exists(Paths.get(dockerConfigProperty))) {
-            return dockerConfigProperty;
-        } else if (Files.exists(dockerConfig)) {
-            return dockerConfig.toString();
-        } else {
-            return null;
-        }
```

Comment:
Exposing docker config.json to the containerised compose now does more harm than good, as now understands, but can't use, the credential helpers.

Instead, remove this config passthrough altogether.